src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -86,6 +86,20 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
 
+                ->arrayNode('api_keys')
```

Comment:
Maybe we should put oauth and this under a `swagger` node. IMO the name is too generic.

Same with the parameter, I think `api_platform.swagger.api_keys` is cleaner then `api_platform.api_keys` which is misleading.